Statistics/Histograms/ThreadsafeInlineHistogram.cs

```diff
@@ -569,6 +569,46 @@ public double InverseCDF(double p)
             }
         }
 
+        public static ThreadsafeInlineHistogram AddHistograms(List<ThreadsafeInlineHistogram> histograms)
+        {
+            if (histograms.Count == 0)
+            {
+                ConvergenceCriteria defaultConvergenceCriteria = new ConvergenceCriteria();
+                return new ThreadsafeInlineHistogram(defaultConvergenceCriteria);
```

Comment:
It is best to avoid multiple return statements. Not the end of the world, but you could probably do something like:
ThreadsafeInlineHistogram histogramToReturn = new ThreadsafeInlineHistogram(defaultConvergenceCriteria); at the top of the method. Then if histograms.count>0 
histogramToReturn = someNewHisto
and then at the bottom return the histogram.

Reply:
done